src/components/tasks/card/card.module.scss

```diff
@@ -49,6 +49,6 @@
 }
 
 .cardStrongFont {
-  font-size: 1.5rem;
+  font-size: 1.3rem; 
```

Comment:
Can we remove extra spaces?
Also, make sure that the `rem` unit converts to whatever font-size suggested in the designs

Reply:
ok